src/TestHelper/dir.props

```diff
@@ -0,0 +1,8 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Is there a more descriptive name than TestHelper we could use?  CoreFx.TestUtilities or something along those lines?